src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/delete_command_request.cc

```diff
@@ -100,6 +100,7 @@ void DeleteCommandRequest::Run() {
     ++chaining_counter;
   }
 
+  // cppcheck-suppress unreadVariable
   if ((*command).keyExists(strings::vr_commands)) {
```

Comment:
What exact variable was not read? Check if we need this suppress

Reply:
Fixed in 91d94fd27998e37ca34e291310b032be76698c44
Please check